source/documentation/appendix-docs/conda.md

```diff
@@ -23,18 +23,26 @@ Most (around 95%) R packages on CRAN are available through `conda`. They have th
 
 #### Examples
 
-##### Installing a package:
+The following tables show conda commands and their base R analogues.
 
+- Installing a package:
```

Comment:
I think it would better without bullet points, especially as the tables will not share the same indentation.
```suggestion
Installing a package:
```